### PR TITLE
Updating core tools to 4.30.0

### PR DESCRIPTION
**Pull request by v-imohammad**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

**Review by VpOfEngineering — APPROVED** [review]

**Comment by khkh-ms** [comment]
I pushed another commit to your's branch's head. However, I couldn't push to your branch.

**Comment by kshyju** [comment]
@v-imohammad Is this still valid/relevant? If so, can you rebase?

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj [hunk 1]
```diff
@@ -273,7 +273,7 @@
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.21.0" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.2.0" />
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage.Internal" Version="1.4.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.28.5" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.30.0" />
```
Comment (mattchenderson) [inline src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:276]: Obsolete. Already ahead: https://github.com/Azure/azure-functions-core-tools/blob/2b7cdb580f3197b97312222a7dc9b304c8e4d9f0/src/Azure.Functions.Cli/Azure.Functions.Cli.csproj#L295

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj [hunk 2]
```diff
@@ -285,7 +285,7 @@
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7.0" Version="4.0.2973" />
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7.2" Version="4.0.3070" />
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7.4" Version="4.0.3085" />
-    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="4.23.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="4.24.0" />
```
Comment (mattchenderson) [inline src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:288]: Obsolete. Already ahead: https://github.com/Azure/azure-functions-core-tools/blob/2b7cdb580f3197b97312222a7dc9b304c8e4d9f0/src/Azure.Functions.Cli/Azure.Functions.Cli.csproj#L309

**Comment by mattchenderson** [comment]
Changes here are obsolete. Closing.

**Pull request closed without merging**